src/lib/lex.mll

```diff
@@ -40,6 +40,8 @@ let keywords =
     ("dim", DIM);
     ("𝔽", COF);
     ("cof", COF);
+    ("∂", BOUNDARY);
+    ("boundary", BOUNDARY);
```

Comment:
In redtt I think we didn't have cool ASCII syntax for boundary. It seems like it might be a bad idea to reserve the word "boundary" (or "bdry" or whatever). Thoughts?

Reply:
it would certainly be a bad idea to reserve *both* but it seems like one or the other might be ok.

Reply:
I think we should just reserve the cool unicode character and not the word, since "boundary" isn't shorter than "i=0\/i=1".